definitions/schedule.py

```diff
@@ -36,65 +36,83 @@
+                    frequency=[3]),
             Dataset("united-kingdom-full",
                     data_dir="subnational/united-kingdom-full/cases",
                     max_wall_clock_minutes=7*24*60
-                    timeout=3*24*60*60),
+                    timeout=3*24*60*60,
+                    frequency=[4]),
             Dataset("united-kingdom-deaths-full",
                     data_dir="subnational/united-kingdom-full/deaths",
                     max_wall_clock_minutes=7*24*60
-                    timeout=3*24*60*60),
+                    timeout=3*24*60*60,
+                    frequency=[4]),
             Dataset("united-kingdom-admissions-full",
                     data_dir="subnational/united-kingdom-full/admissions",
                     max_wall_clock_minutes=7*24*60
-                    timeout=3*24*60*60),
+                    timeout=3*24*60*60,
+                    frequency=[4]),
             Dataset("canada-full", data_dir="subnational/canada-full/cases",
                     max_wall_clock_minutes=7*24*60,
-                    timeout=3*24*60*60),
+                    timeout=3*24*60*60,
+                    frequency=[7]),
             Dataset("germany-full", data_dir="subnational/germany-full/cases",
                     max_wall_clock_minutes=7*24*60,
-                    timeout=3*24*60*60),
+                    timeout=3*24*60*60,
+                    frequency=[9]),
             Dataset("italy-full", data_dir="subnational/italy-full/cases",
                     max_wall_clock_minutes=7*24*60,
-                    timeout=3*24*60*60),
+                    timeout=3*24*60*60,
+                    frequency=[11]),
             Dataset("brazil-full", data_dir="subnational/brazil-full/cases",
                     max_wall_clock_minutes=7*24*60,
-                    timeout=3*24*60*60),
+                    timeout=3*24*60*60,
+                    frequency=[13]),
             Dataset("colombia-full", data_dir="subnational/colombia-full/cases",
                     max_wall_clock_minutes=7*24*60,
-                    timeout=3*24*60*60),
+                    timeout=3*24*60*60,
+                    frequency=[15]),
             Dataset("india-full", data_dir="subnational/india-full/cases",
                     max_wall_clock_minutes=7*24*60,
-                    timeout=3*24*60*60),
+                    timeout=3*24*60*60,
+                    frequency=[17]),
             Dataset("united-states-full",
                     data_dir="subnational/united-states-full/cases",
                     max_wall_clock_minutes=7*24*60,
-                    timeout=3*24*60*60),
+                    timeout=3*24*60*60,
+                    frequency=[19]),
             Dataset("deaths-full", data_dir="national/deaths-full",
                     max_wall_clock_minutes=7*24*60,
-                    timeout=3*24*60*60),
+                    timeout=3*24*60*60,
+                    frequency=[21]),
```

Comment:
We had discussed distributing these over the month, not just spreading them out. Currently it looks like the smallest datasets run early in the month and big ones later (with a day gap between), not distributed over the month by size instead sorted over the month by size. This still leaves the long running big ones able to bunch up and overlap.


What I'm looking for is something like this:
| Rank | Day of Month |
|-------|----------------|
| 1 |  1 |
| 2 | 15 |
| 3 | 8 |
| 4 | 22 |
| 5 | 4 |
| 6 | 11 |
etc